### PR TITLE
Ship Battle

### PR DESCRIPTION
**Pull request by AvetKhach**
It's not final version

### CODE REVIEW

File: src/com/company/Homework3/ShipBattle.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
+package com.company.Homework3;
+
+public class ShipBattle {
+    public static void main(String[] args) {
```
Comment (Artash62062) [inline src/com/company/Homework3/ShipBattle.java]: Its not the final version please finilize it

**Review by Artash62062 — COMMENTED** [review]
Its not a final version

File: src/com/company/Homework3/ShipBattle.java (hunk 1 above)
Comment (AvetKhach) [reply src/com/company/Homework3/ShipBattle.java]: ok

File: src/com/company/Homework3/ShipBattle.java [hunk 2]
```diff
@@ -0,0 +1,156 @@
...
+        while (ship5_1 == ship6 || ship5_1 == ship5_2 || ship5_1 < 8) {
+            ship5_1 = (int) (Math.random() * 11);
+        }
+        while (ship3_2 == ship6 || ship3_2 == ship5_2 || ship3_2 == ship5_1 || ship3_2 == 0 || ((ship3_2 <= ship6 + 1 && ship3_2 >= ship6 - 1) || (ship3_2 <= ship5_2 + 1 && ship3_2 >= ship5_2 - 1))) {
+            ship3_2 = (int) (Math.random() * 11);
+        }
+        while (ship3_1 == ship6 || ship3_1 == ship5_2 || ship3_1 == ship3_2 || ship3_1 == 0
+                || ((ship3_1 <= ship6 + 1 && ship3_1 >= ship6 - 1) || (ship3_1 <= ship5_2 + 1 && ship3_1 >= ship5_2 - 1) || (ship3_1 <= ship3_2 + 1 && ship3_1 >= ship3_2 - 1))) {
+            ship3_1 = (int) (Math.random() * 11);
+        }
```
Comment (Artash62062) [inline src/com/company/Homework3/ShipBattle.java:82]: Too Complex Code is not readable

Comment (AvetKhach) [reply src/com/company/Homework3/ShipBattle.java:82]: Maybe you want me to add comments to the functions?

**Pull request merged**